src/main/java/lotto/domain/model/LottoTickets.java

```diff
@@ -0,0 +1,51 @@
+package lotto.domain.model;
+
+import lotto.domain.generator.LottoNumberGenerator;
+import lotto.util.ErrorMessages;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LottoTickets {
+    private static final int PRICE_PER_TICKET = 1000;
+
+    private final List<Lotto> tickets;
+
+    public static LottoTickets fromAmount(int amount, LottoNumberGenerator generator) {
+        validateAmount(amount);
+        int count = amount / PRICE_PER_TICKET;
+        List<Lotto> generated = new ArrayList<>();
+        for (int i = 0; i < count; i++) {
+            List<Integer> sortedNumbers = generator.generate().stream()
+                    .sorted()
+                    .collect(Collectors.toList());
```

Comment:
```suggestion
        validateAmount(amount);
        int count = amount / PRICE_PER_TICKET;
        List<Integer> sortedNumbers = generator.generate(count);
```
번호 생성 책임을 genreator에게 담당하게 하는 것이 더 좋지 않을까요?